client/core/core_test.go

```diff
@@ -541,24 +550,27 @@ type TXCWallet struct {
 	preRedeem         *asset.PreRedeem
 }
 
-func newTWallet(assetID uint32) (*xcWallet, *TXCWallet) {
+func newTWallet(assetID uint32, crypter *tCrypter) (*xcWallet, *TXCWallet) {
```

Comment:
Nah. Instead of modifying the existing `tCrypter` and forcing all tests to use it, create a new `Crypter`-implementing stub type, and only do the `encPW` stuff in the test that needs it, not in this general purpose constructor.

Reply:
Good point 👍